locale/ca/docs/index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Documentació
+layout: docs.hbs
+labels:
+  lts: LTS
+---
+
+# Sobre la documentació
+
+Existeixen tres tipus de documentació a l'abast en aquest lloc:
+
+* Referència de la API
+* Característiues d'ES6
+* Preguntes freqüents
+
+### Referència de l'API
+
+La [Referència de l'API](/api/) proporciona informació detallada sobre una funció ó un objecte en Node.js. Aquesta
+documentació indica quins arguments acepta un mètode, el valor que retorna aquest mètode i quins errors poden estar
+relacionats al mateix. També indica quins mètodes estàn a l'abast per les diferents versions de Node.js
+
+També descriu els mòduls inclosos que proporciona Node.js, mes no documenta els mòduls que proporciona la comunitat.
+
+<div class="highlight-box">
+    <h4>¿Buscant la referència de l'API per a una versió anterior?</h4>
```

Comment:
In catalan, there is only one question mark at the end of the sentence :)

Reply:
Thanks for review it @jose-lara!
I've change it all your proposals 💯